guide/popular-topics/miscellaneous-examples.md

```diff
@@ -76,6 +76,10 @@ process.on('unhandledRejection', error => console.error('Uncaught Promise Reject
 
 ## Mention prefix
 
+::: tip
+The `escapeRegex` function is used to convert special characters into literal characters by escaping them, so that they don't terminate the pattern within the [Regular Expression](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)!
```

Comment:
The tip was fine were it was, below the codeblock and above the other tip, because here you're mentioning something that hasn't been shown to the reader yet.